### DISCUSSION TITLE
Unable to run cypress tests on GH actions

### DISCUSSION THREAD

**Discussion by mattgodbolt** [cypress-io/cypress#22406] [CI setup]
As of earlier today all my cypress tets are failing on GH actions with various 5xx errors from cloudflare during cypress installation.

An example: https://github.com/compiler-explorer/compiler-explorer/runs/6971777339?check_suite_focus=true

```
npm ERR! The Cypress App could not be downloaded.
npm ERR! 
npm ERR! Does your workplace require a proxy to be used to access the Internet? If so, you must configure the HTTP_PROXY environment variable before downloading Cypress. Read more: https://on.cypress.io/proxy-configuration
npm ERR! 
npm ERR! Otherwise, please check network connectivity and try again:
npm ERR! 
npm ERR! ----------
npm ERR! 
npm ERR! URL: https://download.cypress.io/desktop/9.6.0?platform=linux&arch=x64
npm ERR! Error: Failed downloading the Cypress binary.
npm ERR! Response code: 520
npm ERR! Response message:
```

I variously get `520` or `524` errors from CloudFlare (it seems, from googling).  Is this a known issues?

**Comment by stavares843** [comment]
This seems to be related with the Cloudflare outage that happened last month as described [here](https://blog.cloudflare.com/cloudflare-outage-on-june-21-2022/)

Going manually to the URL wouldn't work as well, so Github Action would also fail trying to download.

When Cloudflare services got restored, issue was solved.
